app/models/Strapi.ts

```diff
@@ -0,0 +1,31 @@
+export interface Single {
+  data: {
+    id: number;
+    attributes: {
+      [key: string]: string;
+    };
+    meta: {
+      [key: string]: string;
+    }
+  };
+}
+
+export interface Link {
+  id: number;
+  url: string;
+  text: string;
+}
+
+export interface DynamicLink {
+  id: number;
+  __component: string;
```

Comment:
Can you please add a comment about what this somewhat non-obvious internal attribute might be used for in practice?

Reply:
added comment